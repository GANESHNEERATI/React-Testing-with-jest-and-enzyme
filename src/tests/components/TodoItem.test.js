import React from "react";
import { shallow } from "enzyme";
import TodoItem from "../../components/TodoItem";

test("should test Todocomponent", () => {
  const wrapper = shallow(<TodoItem todo="Read  News Papers" />);
  expect(wrapper).toMatchSnapshot();
});
