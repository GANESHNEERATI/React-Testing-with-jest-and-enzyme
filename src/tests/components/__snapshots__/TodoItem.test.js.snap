// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Todocomponent 1`] = `
<div
  className="todo-item"
>
  <h2>
    Read  News Papers
  </h2>
</div>
`;
